Content.Server/Botany/SeedPrototype.cs

```diff
@@ -108,13 +108,18 @@ public partial class SeedData
 
     /// <summary>
     ///     If true, the properties of this seed cannot be modified.
+    ///   to spare others like me: this DOES NOT prevent mutations
     /// </summary>
     [DataField("immutable")] public bool Immutable;
 
+    /// <summary>
+    /// If true, you cannot clip this plant for more seeds, used for special plants such as the gnome plant
+    /// </summary>
+    [DataField("unclipable")] public bool Unclipable;
```

Comment:
I would like "Unclipable" changed to "Unclippable" because it sounds more like english

Reply:
aye ill get that changed